<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPortNameDialog</class>
 <widget class="QDialog" name="SetPortNameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>채널 이름</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="editPortName3">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editPortName4">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ch.5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="editPortName5">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Ch.6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editPortName1">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ch.1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Ch.3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Ch.4</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="editPortName6">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Ch.7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ch.2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editPortName2">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="editPortName8">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Ch.8</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="editPortName7">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="editPortName9">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Ch.9</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Ch.13</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Ch.14</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QLineEdit" name="editPortName14">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Ch.12</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Ch.15</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QLineEdit" name="editPortName15">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Ch.11</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="editPortName11">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="editPortName12">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="editPortName10">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Ch.10</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLineEdit" name="editPortName13">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QLineEdit" name="editPortName16">
        <property name="maxLength">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Ch.16</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetPortNameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetPortNameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
